README.md

```diff
@@ -52,6 +52,12 @@ Usage of ./up:
     	A file containing queries to run against the read endpoint.
   -threshold float
     	The percentage of successful requests needed to succeed overall. 0 - 1. (default 0.9)
+  -tls-ca-file string
+    	File containing the TLS CA to use against servers for verification. If no CA is specified, there won't be any verification.
+  -tls-cert-file string
```

Comment:
Typically a flag name for this has `client` somewhere in it, to signal that it's really just about client auth. I'd suggest `-tls-client-cert-file`.